eng/ignore-links.txt

```diff
@@ -0,0 +1,2 @@
+#Place holder for the temporary links
+not-valid-link
```

Comment:
I would remove "not-valid-link" and just leave the comment `# Placeholder file for links that need exceptions from link checking`

Reply:
We have some #note-valid-link in README-TEMPLATE.md which failed the scripts. How do we plan to suppress the check if remove here.

Reply:
Ok we should remove that link from the template.md then and just add a comment, see an example in https://github.com/Azure/azure-sdk/pull/1587/files.